grid2op/tests/test_score_idf_2023.py

```diff
@@ -49,7 +51,9 @@ def act(self, observation: BaseObservation, reward: float, done: bool = False) -
 class TestScoreL2RPN2023(unittest.TestCase):
     
     def setUp(self) -> None:
-        env_name = "l2rpn_case14_sandbox"
+        env_name = os.path.join(
+            PATH_DATA_TEST, "l2rpn_idf_2023_with_alert"
+        )
         with warnings.catch_warnings():
             warnings.filterwarnings("ignore")
             self.env = grid2op.make(env_name,
```

Comment:
It seems that in this test no opponent is played, thus the Alert cannot be "rewarded". (when no attack, alertreward is always 0).